components/Favorites/Favorites.module.css

```
@@ -0,0 +1,21 @@
+
+.favbuttonoff {
```

camelCase? not important though

```suggestion
.favButtonOff {
```